src/citrine/resources/predictor.py

```diff
@@ -196,6 +196,14 @@ def rename(self,
         entity = self.session.put_resource(path, json, version=self._api_version)
         return self.build(entity)
 
+    def generate_feature_effects(self,
+                                 uid: Union[UUID, str],
+                                 *,
+                                 version: Union[int, str] = MOST_RECENT_VER) -> GraphPredictor:
+        path = self._construct_path(uid, version, "shapley/generate")
+        self.session.put_resource(path, {}, version=self._api_version)
+        return self.get(uid, version=version)
```

Comment:
The `FeatureEffects` object returned by the `generate` call will always be in progress and have no results yet. Additionally, as currently constructed, it doesn't provide an easy way to get the updated status: you'd have to retrieve the corresponding predictor, then call `.feature_effects` on it. So instead, this does the get automatically.